pkg/reconciler/kubequota/scoped_generic_shared_informer_factory.go

```diff
@@ -192,8 +192,19 @@ type scopedGenericNamespaceLister struct {
 // List lists all instances from the cache.Indexer scoped to a single logical cluster and namespace, and matching
 // selector.
 func (s *scopedGenericNamespaceLister) List(selector labels.Selector) (ret []runtime.Object, err error) {
```

Comment:
Who else uses this?

Reply:
Nobody yet, but I could see us using it for GC